modules/react-map-gl-draw/src/editor.js

```diff
@@ -556,6 +559,18 @@ export default class Editor extends PureComponent<EditorProps, EditorState> {
     }
   };
 
+  _onPan = (evt: MjolnirEvent) => {
```

Comment:
add a comment about the conditions of panning event not being forward to map.

Reply:
Will do!